components/ui/Loader.tsx

```diff
@@ -108,6 +110,19 @@ export const Loader: React.FC<LoaderProps> = props => {
             </SafeAreaView>
           </Row>
           <View style={Theme.Styles.hrLineFill}></View>
+          {props.showBanner && (
+            <BannerNotification
+              type={
+                props.bannerType
+                  ? props.bannerType
+                  : InjiConstants.BANNER_TYPE_SUCCESS
```

Comment:
If you are working on extracting bannertype as a constant, can you please add this constant even in other banner notification component consumers & in banner notification props too?

This way we can maintain consistency too

Reply:
Updated in all the occurences